Source/WebCore/accessibility/win/AccessibilityObjectWrapperWin.h

```diff
@@ -30,26 +30,30 @@
 
 namespace WebCore {
 
-    class AccessibilityObject;
-
-    class AccessibilityObjectWrapper : public IUnknown {
-    public:
-        // IUnknown
-        virtual HRESULT STDMETHODCALLTYPE QueryInterface(_In_ REFIID riid, _COM_Outptr_ void** ppvObject) = 0;
-        virtual ULONG STDMETHODCALLTYPE AddRef() = 0;
-        virtual ULONG STDMETHODCALLTYPE Release(void) = 0;
-
-        virtual void detach() = 0;
-        bool attached() const { return m_object; }
-        AccessibilityObject* accessibilityObject() const { return m_object; }
-
-        WEBCORE_EXPORT void accessibilityAttributeValue(const AtomString&, VARIANT*);
-
-    protected:
-        AccessibilityObjectWrapper(AccessibilityObject* obj) : m_object(obj) { }
-        AccessibilityObjectWrapper() : m_object(nullptr) { }
-
-        AccessibilityObject* m_object;
-    };
+class AccessibilityObject;
+
+class AccessibilityObjectWrapper : public IUnknown {
+public:
+    // IUnknown
+    virtual HRESULT STDMETHODCALLTYPE QueryInterface(_In_ REFIID riid, _COM_Outptr_ void** ppvObject) = 0;
+    virtual ULONG STDMETHODCALLTYPE AddRef() = 0;
+    virtual ULONG STDMETHODCALLTYPE Release(void) = 0;
+
+    virtual void detach() = 0;
+    bool attached() const { return m_object; }
+    AccessibilityObject* accessibilityObject() const { return m_object; }
+
+    WEBCORE_EXPORT void accessibilityAttributeValue(const AtomString&, VARIANT*);
+
+protected:
+    AccessibilityObjectWrapper(AccessibilityObject* obj)
+        : m_object(obj)
+    { }
+    AccessibilityObjectWrapper()
+        : m_object(nullptr)
+    { }
```

Comment:
The above two constructors are the only other change in this file – changing constructor definition with member initializers from one-line to multiline, so that [member initialization has proper code style](https://webkit.org/code-style-guidelines/#punctuation-member-init).